neo/rawio/blackrockrawio.py

```diff
@@ -1254,7 +1254,18 @@ def __read_nev_data(self, nev_data_masks, nev_data_types):
         # read all raw data packets and markers
         dt0 = [("timestamp", ts_format), ("packet_id", "uint16"), ("value", f"S{data_size - header_skip}")]
 
-        raw_data = np.memmap(filename, offset=header_size, dtype=dt0, mode="r")
+        # expected number of data packets
```

Comment:
```suggestion
        # expected number of data packets. We are not sure why, but it seems we can get partial data packets
        # based on blackrock's own code this is okay so applying an int to round down is necessary to obtain the
        # memory map of full packets and toss the partial packet.
```
what about something like this? So that if we are ever able to track down these partials we know where in the code we made the assumption. I'm thinking that the way the machine organizes data it must just write out partial packets, but since they aren't fully formed the blackrock dev's toss that final partial packet?

Reply:
@zm711 sounds reasonable to me! If you'd like, we could even include a reference to their code:
https://github.com/BlackrockNeurotech/Python-Utilities/blob/fa75aa671680306788e10d3d8dd625f9da4ea4f6/brpylib/brpylib.py#L580-L587

Reply:
That sounds good to me. Is there a way to have the link not be the giant commit blob? I tried copying a simpler link, but it did the same thing to me.  maybe the link is only pretty when it is an issue or PR? 
like if I link to your issue it is a very simple link without the commit and the blob. But maybe I don't usually connect straight to code?
https://github.com/NeuralEnsemble/python-neo/issues/1704

Reply:
not that I'm aware of, if you want to reference a specific piece of code

Reply:
That's what I thought too, but I was hoping :/ Such is life. I'm fine either with or without the link. Whatever you prefer.